Makefile

```diff
@@ -19,6 +19,10 @@ test:
 	go test -race -v ./...
 	go vet ./...
 
+static:
```

Comment:
Add this to `.PHONY` targets, or this target does nothing because of `static` directory.